File: insercionarray/inser2/inser.js
let inputArr = [11, 21, 31, 41, 51, 61];
let num = 52;

let antes = "Antes de la insercion, el array es: ";
for (let i = 0; i < inputArr.length; i++) {
    antes += inputArr[i] + " ";
}
console.log(antes);

inputArr.splice(5, 0, num);

let despues = "Despues de la insercion, el array es: ";
for (let i = 0; i < inputArr.length; i++) {
    despues += inputArr[i] + " ";
}
console.log(despues);

File: insercionarray/inser2/inser.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> inputArr = {11, 21, 31, 41, 51, 61};
    int num = 52;

    cout << "Antes de la insercion, el array es: ";
    for (int i = 0; i < inputArr.size(); i++) {
        cout << inputArr[i] << " ";
    }

    inputArr.insert(inputArr.begin() + 5, num);

    cout << "\nDespues de la insercion, el array es: ";
    for (int i = 0; i < inputArr.size(); i++) {
        cout << inputArr[i] << " ";
    }

    return 0;
}